### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by codeboten**
Updating readme and renaming package:
- updating references to ddtrace-run to use ls-trace-run
- setup.py updated 

Signed-off-by: Alex Boten <aboten@lightstep.com>

### CODE REVIEW

**Comment by tedsuo** [comment]
This is spot on. I haven’t reviewed in detail, but this is what I was looking for. 

Have the commands been tested for copy-pastability? Are there changes we can make which reduce the need for configuration when using this tracer with LightStep?

**Review by tedsuo — APPROVED** [review]
Please get a second reviewer. But I am fine with this being merged as is.

**Comment by codeboten** [comment]
@tedsuo: commands have been tested for pastability. Not sure there's much more left to trim as far as configuration does, the steps are:
- pip install package
- export 2 env variables
- run the app via the tracer runtime

File: README.md [hunk 1]
```diff
@@ -1,90 +1,66 @@
...
 
+A trace from the application should be available in your [LightStep dashboard](https://app.lightstep.com/)
 
-#### Configuration
+## Next Steps
 
-The CI tests are configured through [config.yml](.circleci/config.yml).
+Check out https://docs.lightstep.com/docs/python-auto-instrumentation for more information
 
+## Support
```
Comment (mwear) [inline README.md:65]: Do you want to mention the relationship between the ls-trace version and dd-trace version?

**Review by mwear — APPROVED** [review]
Just the one comment about the version. Otherwise, 👀 good!

File: README.md (hunk 1 above)
Comment (codeboten) [reply README.md:65]: done!

**Pull request merged**